run.bat

```diff
@@ -0,0 +1 @@
+python3 main.py
```

Comment:
The installer should be the one to create this executable file. If the game isn't installed then only an install script should be available.

Reply:
Correct, this is not a good fix, I will close it.